### PR TITLE
fix: parsing for kms tests

### LINKED ISSUE #412: kms tests fail when Werelogs output is changed

### PR DESCRIPTION
**Pull request by rahulreddy**
The output parsing was heavily reliant on the order of fields, so a new change in Werelogs
would break the tool. The parsing has been updated to accommodate future changes.
Fix #412

### CODE REVIEW

**Comment by rahulreddy** [comment]
@ironman-machine

File: tests/functional/aws-node-sdk/test/object/copyPart.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ function createEncryptedBucket(bucketParams, cb) {
     const child = childProcess.spawn(args[0], args)
     .on('exit', () => {
         const hasSucceed = body.join('').split('\n').find(item =>
-            item.startsWith('{"name":"S3","statusCode":200'));
+            item.indexOf('"statusCode":200')) !== -1;
```
Comment (ploki) [inline tests/functional/aws-node-sdk/test/object/copyPart.js]: why don't you parse the json output instead of doing string matching?

Comment (rahulreddy) [inline tests/functional/aws-node-sdk/test/object/copyPart.js]: I could.

**Comment by rahulreddy** [comment]
Updated the parsing to parse into JSON. I haven't moved them into a common utility as they have different setup and I did not want to get into a refactor zone.

**Comment by rahulreddy** [comment]
@ironman-machine

**Comment by rahulreddy** [comment]
@ironman-machine ...

**Pull request merged**